app/views/search/_result.html.erb

```diff
@@ -61,6 +61,12 @@
       </div>
     <% end %>
 
+    <% if result.db_source.present? %>
+      <p class="result-pubinfo">
```

Comment:
I used an existing class because this is kinda temp and I know I'd forget to remove any custom CSS classes I made when this gets axed.